webpack.config.js

```diff
@@ -81,6 +107,9 @@ module.exports = (env = process.env.NODE_ENV || 'production') => ({
           WHISTLER_API_SERVICE_URL: 'https://whistler.onint.ciscospark.com/api/v1'
         })
       ] : [
+        new BannerPlugin({
+          banner: `Webex JS SDK v${process.env.VERSION || version}`
```

Comment:
Do you wanna add a copyright notice?

Reply:
Yup
https://unpkg.com/browse/react@16.13.1/umd/react.production.min.js

Reply:
It's also easier to track which version of the SDK you're using, if you're just pulling latest